__tests__/commands/_helpers.js

```diff
@@ -38,7 +38,7 @@ export async function createLockfile(dir: string): Promise<Lockfile> {
     lockfile = parse(rawLockfile).object;
   }
 
-  return new Lockfile(lockfile);
+  return new Lockfile(lockfile, undefined, 'success');
```

Comment:
The third parameter is just the parse result type, not the parse result itself. May be I should rename the variable to make that clear?